sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/OpenTelemetryAssertions.java

```diff
@@ -41,6 +42,11 @@ public static SpanDataAssert assertThat(@Nullable SpanData spanData) {
     return new SpanDataAssert(spanData);
   }
 
+  /** Returns an assertion for {@link StatusData}. */
+  public static StatusDataAssert assertThat(@Nullable StatusData statusData) {
+    return new StatusDataAssert(statusData);
+  }
```

Comment:
I don't think we should add a top level assertion for `StatusData`. These assertions will always be done in the context of asserting against a particular span.